benefits/static/css/styles.css

```diff
@@ -81,6 +81,19 @@ li {
   margin: 0;
 }
 
+/* First p immediately after an H1 */
+h1 + p:first-of-type,
+h1 + .media-list,
+/* Last p sibling to an H1 */
+.media-body--details p:not(:first-of-type) {
+  padding-top: 24px;
+}
+
+/* Last p siblings to an H1 */
+h1 ~ p:nth-last-of-type(1) {
+  padding-bottom: 24px;
+}
+
```

Comment:
These are cool selectors to get this styling in CSS... I wonder if:

1. We could wrap these up into helper classes and somehow apply them at the correct time / template to be more declarative?
2. In absence of that, maybe at least add some comments here as to which pages these rules are targeting. E.g. it wasn't obvious to me where the `h1 + .media-list` combination actually occurs.

Reply:
1. There is no way to use adjacent/sibling-selector logic in a class unfortunately.
2. I will add comments on where these are used